lib/Doctrine/ORM/Tools/Console/ConsoleRunner.php

```diff
@@ -91,13 +91,16 @@ public static function createApplication(HelperSet $helperSet, array $commands =
      */
     public static function addCommands(Application $cli) : void
     {
-        $cli->addCommands(
-            [
+        if (class_exists(DBALConsole\Command\ImportCommand::class)) {
```

Comment:
Should the ORM expose DBAL console commands? The new commands are not really compatible, not sure how to make them.